tools/download-install.sh

```diff
@@ -6,6 +6,22 @@ set -e
 ## Usage: ./download-install.sh                            --> installs the latest version
 ##        TERRA_CLI_VERSION=0.1.0 ./download-install.sh    --> installs version 0.1.0
 
+readonly REQ_JAVA_VERSION=17
+echo "--  Checking if installed Java version is ${REQ_JAVA_VERSION} or higher"
+if [[ -n "$(which java)" ]];
+then
+  # Get the current major version of Java: "11.0.12" => "11"
+  readonly CUR_JAVA_VERSION="$(java -version 2>&1 | awk -F\" '{ split($2,a,"."); print a[1]}')"
+  if [[ "${CUR_JAVA_VERSION}" -lt ${REQ_JAVA_VERSION} ]];
+  then
+    echo "Error with Java version: set to ${CUR_JAVA_VERSION}, exiting..."
```

Comment:
The info message above indicates what is being checked (java version minimum), but it's a good practice to have the error as informative as possible in case the end user gets only one line of information.

```
echo "ERROR: Java version detected (${CUR_JAVA_VERSION}) is less than required (${REQ_JAVA_VERSION})
```

I also like to send errors to stderr instead of stdout, though it isn't done in this file. I'd prefer to see all of the error messages be written as:


```
>&2 echo "ERROR: Java version detected (${CUR_JAVA_VERSION}) is less than required (${REQ_JAVA_VERSION})"
```

Reply:
Done update all files in this repo